### DISCUSSION TITLE
Updated weights

### DISCUSSION THREAD

**Discussion by rs2-davebanguilan** [vladmandic/face-api#122] [Q&A] [answered]
Hi, I just want to ask if you guys have other updated weights aside from the [original model weights](https://github.com/justadudewhohacks/face-api.js-models). Thanks

**Comment by vladmandic** [comment, marked as answer]
`face-api` is in maintenance model, there are no new models.  
all the new development is going towards my new library which supersedes `face-api`: <https://github.com/vladmandic/human>

**Reply by rs2-davebanguilan** [reply]
Thank you so much @vladmandic
